tests/RA_StringUtils_Tests.cpp

```diff
@@ -285,11 +285,15 @@ TEST_CLASS(RA_StringUtils_Tests)
         std::string input("This is a test.");
         const Tokenizer tokenizer(input);
         const auto* pStart = input.c_str();
-        Assert::AreEqual(0, tokenizer.GetPointer(0U) - pStart);
-        Assert::AreEqual(4, tokenizer.GetPointer(4U) - pStart);
-        Assert::AreEqual(8, tokenizer.GetPointer(8U) - pStart);
-        Assert::AreEqual(12, tokenizer.GetPointer(12U) - pStart);
-        Assert::AreEqual(15, tokenizer.GetPointer(16U) - pStart); // string is 15 characters
+
+        // ptrdiff_t is 8 bytes but a ToString overload isn't working
+        Assert::AreEqual(0, gsl::narrow_cast<int>(tokenizer.GetPointer(0U) - pStart));
+        Assert::AreEqual(4, gsl::narrow_cast<int>(tokenizer.GetPointer(4U) - pStart));
+        Assert::AreEqual(8, gsl::narrow_cast<int>(tokenizer.GetPointer(8U) - pStart));
+        Assert::AreEqual(12, gsl::narrow_cast<int>(tokenizer.GetPointer(12U) - pStart));
+
+        // TODO: This is an error in logic and should not be considered success
+        Assert::AreEqual(15, gsl::narrow_cast<int>(tokenizer.GetPointer(16U) - pStart)); // string is 15 characters
```

Comment:
this is validating intentional behavior. if you ask for a pointer beyond the end of the string, the end of the string should be returned.

Reply:
Alright, I'll get rid of that comment when I get a chance.